### ISSUE TITLE
Install rancher on Raspberry Pi

### ISSUE THREAD

**Issue by markdav-is** [OpenEugene/cloud-in-a-box#7]
Is it even possible? https://rancher.com/quick-start/

**Comment by colindavey** [comment]
According to the above link, Rancher calls for a 64-bit Linux. The Raspberry Pi has 64-bit hardware, but the 64-bit version of the Raspberry Pi OS is still in development, and not stable yet. (https://raspberrytips.com/raspberry-pi-os-64-bits-vs-32-bits/)

Here (https://rancher.com/docs/os/v1.x/en/installation/server/raspberry-pi/) it says there is a Raspberry Pi image of Rancher, but only Raspberry Pi 3 is tested and known to work. The one I have is RP 4. 

However, this seems promising: https://ubuntu.com/download/raspberry-pi.

**Comment by colindavey** [comment]
OK, got the right versions of Ubuntu and Docker installed (long story, I'll document it some time) and then went to the "second easy step" `To install and run Rancher, execute the following Docker command on your host:...`, and got 
```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform 
(linux/arm64/v8) and no specific platform was requested
``` 

I couldn't find a way to request a specific platform, and there's also this:
```
Important:
Running on an ARM64 platform is currently an experimental feature and is not yet officially supported in 
Rancher. 
Therefore, we do not recommend using ARM64 based nodes in a production environment.
```
https://rancher.com/docs/rancher/v2.x/en/installation/resources/advanced/arm64-platform/

Also, I didn't understand this:
```
To access the Rancher server UI, open a browser and go to the hostname or address where the container 
was installed. You will be guided through setting up your first cluster.
```
(https://rancher.com/quick-start)
In other words, I'm not sure what url to try. I tried `http://localhost:443` and `http://localhost:80`. 

Probably no urls would work at this point, given the warning about the wrong platform, but it's not clear which one to try.

**Comment by colindavey** [comment]
I thought it would be useful to Zoom from the RP, and share screens while trying things. However, Zoom doesn't seem to support the ARM platform.
https://devforum.zoom.us/t/have-you-considered-building-zoom-for-the-raspberry-pi-4-linux-system/14257/13

**Comment by markdav-is** [comment]
I picked up an R-Pi from my place in Eugene today.  It might not have enough RAM, But I have an 8GB one coming.  We really don't need Rancher, but it would be cool to try.  We can get away with just Oqtane on the R-Pi but then we have Blazor lock-in.

**Comment by markdav-is** [comment]
This is a good topic to bring up in the MVP Studio Slack channel.  When I use Rancher at MVP, it's at port 8443.  example: https://100.64.0.1:8443/g/clusters

**Comment by markdav-is** [comment]
If you would prefer an R-Pi3 I can send you one of those as well.  I have a 3 and can try that image you found.

**Comment by colindavey** [comment]
If Rancher doesn't work, we can still do some form of k8s, no doubt. 

https://www.hanselman.com/blog/how-to-build-a-kubernetes-cluster-with-arm-raspberry-pi-then-run-net-core-on-openfaas
https://alexellisuk.medium.com/five-years-of-raspberry-pi-clusters-77e56e547875

These guys (including our old friend Hanselman) seem to be about clustering several RPs together. 

I bet the MVP folks could provide helpful hints about many aspects of the project.

**Comment by colindavey** [comment]
Docker works, btw. The install went smoothly, and I ran the Docker Hello World program.
